VehicleCode/TestingAndCalibration/ReceiveTesting/src/main.cpp

```diff
@@ -75,12 +75,11 @@ void setup() {
         Serial.println(" ms.");
         Serial.println("Reading on CAN2.");
         Serial.println("Starting.");
-        car.setLogState(true);
+        //car.setLogState(true);
     }
 }
 
 
 void loop() {
     car.readSensors();
-    delay(DELAYBY);  
```

Comment:
Please leave this and set DELAYBY to zero when you don't want to delay. It makes debugging easier